doc/index.md

```diff
@@ -1167,6 +1168,67 @@ arcgis.item(itemId)
 })
 ```
 
+### item.data
+
+Get the data content of the item such as WebMap or App configuration.
+
+**Params:**
+String of an Item ID.
+
+**Returns:**
+Promise that resolves JSON Object with item data.
+
+###### **Example**
+
+```
+arcgis.item(itemId)
+.then(function (item) {
+  return item.data()
+})
+.then(function (data) {
+  console.log(data)
+})
+```
+
+### item.create
+
+> Creates a new Item and optional data
+
+**Params:**
+JSON Options object
+
+| Options          | Default   | Description             |
+| ---------------- | --------- | ----------------------- |
+| title            | none      | String. Name of the group
+| description      | none      | String. Description of the group
+| snippet          | none      | String. > 256 character summary
+| tags             | none      | Array. tags for group
+| access           | 'private' | String. 'private', 'public', or 'org'
+| url              | none      | String. Web URI
+| data             | none      | String. Data content such as WebMap JSON or App configuration
+| licenseInfo      | none      | String. access and usage permissions or constraints
+| thumbnail        | none      | String. URL to a thumbnail
+| extent           | none      | String. minimum bounding extent
+| type             | none      | String. Type of item from all Portal types
+
+
+###### **Example:**
+
+```
+var options = {
+  title: 'My New Item',
+  description: 'This item is both new and mine',
+  tags: ['my', 'new', 'cool', 'item'],
+  type: 'Web Mapping Application',
+  access: 'public',
```

Comment:
> [ref:](http://resources.arcgis.com/en/help/arcgis-rest-api/index.html#/Add_Item/02r30000008q000000/) The item is flagged as private (that is, not shared with any group) by default. This can be changed using the [Share Item operation](http://resources.arcgis.com/en/help/arcgis-rest-api/02r3/02r30000007s000000.htm).

since its not possible to updating permissions until *after* the item has been created, we probably shouldn't include `access` in the example code snippet.